test/replication.rest.test.js

```diff
@@ -139,12 +139,17 @@ describe('Replication over REST', function() {
         setAccessToken(peterToken);
       });
 
+      var pushSince = -1;
+      var pullSince = -1;
+
       it('allows pull from server', function(done) {
-        RemoteCar.replicate(LocalCar, function(err, conflicts, cps) {
+        RemoteCar.replicate(LocalCar, pullSince, function(err, conflicts, cps) {
           if (err) return done(err);
 
           if (conflicts.length) return done(conflictError(conflicts));
 
+          pullSince = cps;
```

Comment:
Sharing state between tests is evil. When the first test fails, the second fails too because the first test did not leave the environment in the state expected by the second test.

Please rework your two new tests to not rely on the outcome of previous tests.